packages/reactivity/src/reactive.ts

```diff
@@ -241,9 +241,9 @@ export function toRaw<T>(observed: T): T {
   return raw ? toRaw(raw) : observed
 }
 
-export function markRaw<T extends object>(
-  value: T
-): T & { [RawSymbol]?: true } {
+export type Raw<T> = T & { [RawSymbol]?: true }
```

Comment:
```suggestion
export type MarkRaw<T> = T & { [RawSymbol]?: true }
```

To be more specific?

Reply:
Done.